app/models/user.rb

```diff
@@ -7,11 +7,16 @@ class User < ApplicationRecord
 
   validates :fb_id, uniqueness: true, presence: true #this can be removed when some other signup/login method is added.
   validates :email, uniqueness: true
+  validates_presence_of :first_name, :last_name
 
   def badges
     donated_causes.uniq
   end
 
+  def name
```

Comment:
I know it seems silly, but I think we should write a spec for `name`. I believe all public methods should be specced. There are others on this model that snuck past, but those are outside the scope of this P.R.

Reply:
You're right. It skipped my mind. Thanks for pointing it out. I'll take care of it.